example/src/ThemeLoader.js

```diff
@@ -0,0 +1,83 @@
+import cssVarsPonyfill from 'css-vars-ponyfill';
+
+const THEME_URL_PARAM ='themeId';
+const THEME_CDN_URL   = './';
+
+class ThemeLoader {
+
+
+    static load() {
+
+        return Promise.all([
+            ThemeLoader.applyThemeIfExists(),
+            ThemeLoader.startCssVarsPonyfill(),
+        ]);
+
+    }
+
+
+    /**
+     * Loads the CSS ponyfill.
+     *
+     * In case the property `onlyLegacy` is set to true, it will run only in legacy browsers like IE11.
+     * For test purposes, it's set to true, so can be tested on modern browsers.
+     */
+    static startCssVarsPonyfill() {
```

Comment:
What if you're not using the polyfill, or it's not loaded because you're on a modern browser?

Reply:
It has the flag `{ onlyLegacy: true  | false }` so it's not doing anything if a browser support css vars